tests/FSharp.AWS.DynamoDB.Tests/MetricsCollectorTests.fs

```diff
@@ -281,11 +271,9 @@ type ManyReadOnlyItemsFixture() =
     let table = base.CreateEmpty<MetricsRecord>()
 
     let hk = guid ()
-
     do
```

Comment:
for these, I'm wondering if the `do` can go on the same line, or does it separate it if you do (very minor of course)